solvers/H3LAPD/EquationSystems/HW2DSystem.hpp

```diff
@@ -1,5 +1,5 @@
-#ifndef H3LAPD_HW2DIN3D_SYSTEM_H
-#define H3LAPD_HW2DIN3D_SYSTEM_H
+#ifndef H3LAPD_HW2D_SYSTEM_H
```

Comment:
Not mandatory for this PR because of precedent: I think we should get into the habit of prefixing our include guards with `_NESO_`, all the NP ones are `_NESO_PARTICLES_` etc.

Reply:
Done!